ORStools/proc/directions_lines_proc.py

```diff
@@ -167,7 +174,10 @@ def _get_sorted_lines(layer, field_name):
 
         for feat in sorted(layer.getFeatures(), key=lambda f: f.id()):
             line = None
-            field_value = feat[field_name]
+            try:
+                field_value = feat[field_name]
+            except TypeError:
+                field_value = None
```

Comment:
do we need a try here? (or also just if else as above)
```suggestion
            field_value = feat[field_name] if field_name else None
```

Reply:
no, we don't. fixed that

Reply:
not yet fixed in that file

Reply:
and done :)